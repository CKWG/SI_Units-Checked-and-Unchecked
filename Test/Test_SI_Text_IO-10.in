12345678901    Unit to use
    1980.0*m
   1983.0*m
  1995.0*m
-2005.0*km     m
  2012.0/sVXXX kHz
  2022.0*A     mA
 987654*Om     MOhm
987654*Ohm     /uS
    98765_
98765A*Ohm
+8*mmol
