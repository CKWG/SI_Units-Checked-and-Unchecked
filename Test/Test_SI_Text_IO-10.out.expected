1980.000
SI.ILLEGAL_UNIT
1995.000*m
-2005000.000*m
 2.012*kHz
2021999.875*mA
SI.ILLEGAL_UNIT
 0.988/uS
ADA.IO_EXCEPTIONS.DATA_ERROR
ADA.IO_EXCEPTIONS.DATA_ERROR
 0.008*mol
